apps/backend/src/main/java/tabletop/gather/backend/plan/PlanService.java

```diff
@@ -20,73 +20,138 @@ public class PlanService {
   private final UserRepository userRepository;
   private final GameRepository gameRepository;
 
-  private final GatheringRepository gatheringRepository;
-
   public PlanService(final PlanRepository planRepository, final UserRepository userRepository,
-                     final GameRepository gameRepository, final GatheringRepository gatheringRepository) {
+      final GameRepository gameRepository) {
     this.planRepository = planRepository;
     this.userRepository = userRepository;
     this.gameRepository = gameRepository;
-    this.gatheringRepository = gatheringRepository;
   }
 
-  public List<PlanDto> findAll() {
-    final List<Plan> plans = planRepository.findAll(Sort.by("id"));
+  public List<OverviewPlanDto> findAll() {
+    final List<Plan> plans = planRepository.findAllByIsPrivateFalse();
+    return plans.stream()
+        .map(plan -> mapToDto(plan, new OverviewPlanDto()))
+        .toList();
+  }
+
+  public List<OverviewPlanDto> findAll(UUID userId) {
+    final List<Plan> plans = planRepository.findAllByUserId(userId);
     return plans.stream()
-      .map(plan -> mapToDTO(plan, new PlanDto()))
-      .toList();
+        .map(plan -> mapToDto(plan, new OverviewPlanDto()))
+        .toList();
+  }
+
+  public DetailPlanDto getDetail(final UUID id) {
+    return planRepository.findById(id)
+        .map(plan -> mapToDto(plan, new DetailPlanDto()))
+        .orElseThrow(NotFoundException::new);
   }
 
   public PlanDto get(final UUID id) {
     return planRepository.findById(id)
-      .map(plan -> mapToDTO(plan, new PlanDto()))
-      .orElseThrow(NotFoundException::new);
+        .map(plan -> mapToDto(plan, new PlanDto()))
+        .orElseThrow(NotFoundException::new);
   }
 
-  public UUID create(final CreatePlanDto planDTO, final UUID userId) {
+  public UUID create(final CreatePlanDto planDto, final UUID userId) {
     final Plan plan = new Plan();
-    mapToEntity(planDTO, plan);
+    mapToEntity(planDto, plan);
     plan.setUser(userRepository.findById(userId)
-      .orElseThrow(() -> new NotFoundException("user not found")));
+        .orElseThrow(() -> new NotFoundException("user not found")));
```

Comment:
Mängisch isches `.orElseThrow(() -> new NotFoundException("user not found"))` u mängisch `.orElseThrow(NotFoundException::new)` - ä Backend Cleanup Task wär guet mit däm schomau drufe ;)